### PR TITLE
Remove creation of example data via Flyway

### LINKED ISSUE #45: Remove creation of example data via Flyway
As discussed in #43, we want to remove the creation of example data via Flyway. The first client, user and group, etc. should be created programmatically on the "first start", e.g. when OSIAM starts and the database is empty. Obviously needs #44 to be merged first.

### PR DESCRIPTION
**Pull request by tkrille**
Example data will be created by a special class.
It checks whether the database already contains data and if not,
it creates the example data on startup.

Resolves #45

### CODE REVIEW

File: src/main/java/org/osiam/ExampleData.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+@Component
+public class ExampleData {
+
+    public static final Logger logger = LoggerFactory.getLogger(ExampleData.class);
```
Comment (dacrome) [inline src/main/java/org/osiam/ExampleData.java]: private

**Comment by dacrome** [comment]
Please update changelog

**Comment by tkrille** [comment]
Updated

**Comment by dacrome** [comment]
Thank you!

**Pull request merged**